app/controllers/application_controller.rb

```diff
@@ -2,6 +2,7 @@
 
 class ApplicationController < ActionController::Base
   self.responder = ApplicationResponder
+  include Pundit
```

Comment:
show not authorized page
https://github.com/elabs/pundit#rescuing-a-denied-authorization-in-rails